src/worker/feature-extractor.worker.ts

```diff
@@ -1,9 +1,16 @@
-import { extractFeatures } from "../main/feature-extractor";
+import {
+  extractFeatures,
+  extractFeaturesObservable
+} from "../main/feature-extractor";
 
 onmessage = async function({ data }) {
-  const [buffers, features, extractionParameters] = data;
+  const [buffers, features, extractionParameters, internalParameters] = data;
 
-  postMessage(await extractFeatures(buffers, features, extractionParameters));
+  const batchSize = internalParameters.batchSize || buffers.length;
```

Comment:
This PR isn't mergeable until this breaking change disappears.

Reply:
I've forgotten exactly why this is a breaking change, the tests pass. The tests probably aren't good enough.

Reply:
Also, we're in alpha, we don't care about no stinkin breaking changes!